201-vnet-transitive-bgp/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "sharedKey": {
+            "value": "mysharedkey1234"
```

Comment:
replace this with GEN-UNIQUE to prevent accidental deployments with this value...